Resources/Prototypes/Corvax/Entities/Structures/Machines/printer.yml

```
@@ -47,3 +47,14 @@
       - Document
     storage:
       PrinterPaper: 0
+  - type: DocumentPrinter 
+  - type: ContainerContainer
+    containers:
+      id: !type:ContainerSlot
```

```suggestion
  - type: ContainerContainer
    containers:
      machine_board: !type:Container
      machine_parts: !type:Container
      blueprint: !type:Container
      id: !type:ContainerSlot
```
Наследуемые компоненты перезаписываются